pkgs/applications/misc/qlandkartegt/default.nix

```diff
@@ -0,0 +1,59 @@
+{ mkDerivation, lib, fetchurl, cmake
+, qtmultimedia, qtserialport, qtscript, qtwebkit
+, garmindev, gdal, gpsd, libdmtx, libexif, libGLU, proj }:
+
+mkDerivation rec {
+  name = "qlandkartegt-${version}";
+  version = "1.8.1";
+
+  src = fetchurl {
+    url = "https://bitbucket.org/maproom/qlandkarte-gt/downloads/${name}.tar.gz";
+    sha256 = "1rwv5ar5jv15g1cc6pp0lk69q3ip10pjazsh3ds2ggaciymha1ly";
+  };
+
+  patches = [
+    ./fix-gps_read.patch
```

Comment:
What is the source of these patches? If they are available for direct download it would preferential to use `fetchpatch`.

Reply:
I just don't know, is it OK to fetchpatch from Archlinux repo?

Reply:
Yes, absolutely.

Reply:
Fixed.